install.sh

```diff
@@ -8,8 +8,17 @@ PS4='+ $(date -u -I"seconds" | cut -c1-19) '
 KERNEL_NAME=$(uname -r)
 LOG_FILE_NAME="/var/log/nvidia-installer-$(date +%s).log"
 
+set +euo pipefail
+open_devices="$(lsof /dev/nvidia* 2>/dev/null)"
```

Comment:
debug only but this often helps point to reason for failure (this code doesn't properly handle reinstalls)